weasyl/define.py

```diff
@@ -1033,7 +1038,7 @@ def metric(*a, **kw):
 
 
 def iso8601(unixtime):
-    if isinstance(unixtime, arrow.Arrow):
+    if isinstance(unixtime, arrow.Arrow) or isinstance(unixtime, datetime.datetime):
```

Comment:
Hm, it’s a bit confusing that `created_at` (good name change!) gets put into fields called `"unixtime"` that can end up here. I think we can sort it out later, though, just by looking at every instance of “unixtime”. (Not a change request :)

Reply:
Ya, I was avoiding trying to make too many template changes with this, so I tried to leave most of the unixtime being passed around. Figure a cleanup sweep would be a good idea after we kill all the unixtime instances?